packages/test/end-to-end-tests/package.json

```diff
@@ -119,41 +119,41 @@
     "moniker": "^0.1.2",
     "nock": "^10.0.1",
     "nyc": "^15.0.0",
-    "old-aqueduct": "npm:@fluidframework/aqueduct@^0.33.0-0",
```

Comment:
Are there other prerelease dependency in the rest of the client packages still (e.g. dpendency to common-utils and servers?  Should we just update all of it in one change?

Reply:
Not sure, does the script do this?

Reply:
I don't think I implement a script to automatically do it yet.  probably just want to go thru what you have released and either search and replace, or use bump-version -d

---
In reply to: [563002425](https://github.com/microsoft/FluidFramework/pull/4897#discussion_r563002425) [](ancestors = 563002425)